R/error-checks.R

```diff
@@ -33,6 +33,14 @@ validateIsString <- function(object) {
   validateIsOfType(object, "character")
 }
 
+validateIsNumeric <- function(object) {

```

Comment:
@PavelBal I have added those guys because I am fed up of duplicating the same code all the time